src/coherence/client.py

```diff
@@ -743,6 +773,18 @@ async def remove_map_listener(self, listener: MapListener[K, V], listener_for: O
         else:
             await self._events_manager._remove_key_listener(listener, listener_for)
 
+    @_pre_call_cache
```

Comment:
The docs for extractor state it must not by null.  Should be `None` for Python.  Also, I don't see code that enforces this.